src/verify.v2.integration.test.ts

```diff
@@ -700,23 +682,23 @@ describe("verify(integration)", () => {
     // Ethers would return INVALID_ARGUMENT, as merkle root is odd-length which we tampered it by removing the last char
     expect(isValid(results, ["DOCUMENT_INTEGRITY"])).toStrictEqual(false);
     expect(isValid(results, ["DOCUMENT_STATUS"])).toStrictEqual(false);
-    expect(isValid(results, ["ISSUER_IDENTITY"])).toStrictEqual(true);
+    expect(isValid(results, ["ISSUER_IDENTITY"])).toStrictEqual(false);
```

Comment:
This test is probably the most interesting change that happens in the MR. The identity is not checked anymore, because the signature is not valid.

Reply:
should probably add this in the changelog

Reply:
related to above comment, dyt we should fix it so that if key correct but value malformed it should error? can be confusing to troubleshoot for new implementers if its just skipped and they dont know why